apps/tauri/src/index.css

```
@@ -11,6 +11,8 @@
 /* Tell Tailwind v4 where to scan for utility classes */
 @source "../../../packages/ui/src";
 @source "../../../packages/interface/src";
+@source "../node_modules/@spacedrive/primitives/dist";
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)